pkg/cmd/root/root.go

```diff
@@ -84,8 +85,8 @@ func NewCmdRoot(f *utils.Factory, version string) *cobra.Command {
 
 	// Add subcommands
 	cmd.AddCommand(versionCmd.NewCmdVersion(f))
-	cmd.AddCommand(sensorCmd.NewSensorCmd(f))
-	cmd.AddCommand(authCmd.NewAuthCmd(f))
+	cmd.AddCommand(sensor.NewSensorCmd(f))
+	cmd.AddCommand(auth.NewAuthCmd(f))
```

Comment:
We should come up with a standard. Currently `auth` is importing `authConfigCmd` and now root is doing something else. It is also importing commands differently to deal with the version package conflict.

Obv, using `cmdNameCmd` format doesn't add any functionality (neither does not using it) and the program still works, but neither does having `pkg`, `internal`, and `cmd` paths. However it is a standard we follow because it helps.

So we should agree on something and stick with it. w/e that is @redhatrises @carlosmmatos 

Thanks for coming to my ted talk lol